controllers/submissions.js

```diff
@@ -0,0 +1,23 @@
+const Submission = require('../models/submission');
+
+module.exports = {
+    index: async (req, res, next) => {
+        const submissions = await Submission.find({});
+        res.status(200).json(submissions);
+    },
+    getSubmission: async (req, res, next) => {
+        const { submissionId } = req.value.params;
+        const submission = await Submission.findById(submissionId);
+        res.status(200).json(submission);
+    },
+    getTaskSubmissions: async (req, res, next) => {
```

Comment:
This should be on controllers/tasks

const { taskId } = req.value.params;
const task = await Task.findById(taskId).populate('submissions');
res.status(200).json(task.submissions);

Reply:
done